platform/machine/do/machine.go

```diff
@@ -77,6 +77,11 @@ func (dm *machine) Destroy() {
 }
 
 func (dm *machine) ConsoleOutput() string {
-	// DigitalOcean provides no API for this
-	return ""
+	// DigitalOcean provides no API for retrieving ConsoleOutput
+	// return the journal instead to allow for error checks to be run.
+	data, err := dm.journal.ReadJournal()
```

Comment:
`Destroy()` guards against `dm.journal == nil`, and `ConsoleOutput` is called from `dm.cluster.DelMach()`, so this will need the guard as well.